app/src/main/java/ch/epfl/sdp/ui/main/attending/AttendingEventAdapter.java

```diff
@@ -28,10 +31,20 @@ public AttendingEventViewHolder(@NonNull View itemView) {
         }
     }
 
+    /**
+     * Constructor of the AttendingEventAdapter
+     *
+     * @param attendingEvents a list of events someone attend to
```

Comment:
probably better to change the description of the parameter, I assume this means we get the list of events the user is attending but I am not sure